core/src/main/java/org/apache/brooklyn/util/core/task/BasicExecutionManager.java

```diff
@@ -757,9 +758,23 @@ protected void internalBeforeStart(Map<?,?> flags, Task<?> task) {
             PerThreadCurrentTaskHolder.perThreadCurrentTask.set(task);
             ((TaskInternal<?>)task).setStartTimeUtc(System.currentTimeMillis());
         }
+
+        jitterThreadStart(task);
+
         invokeCallback(flags.get("newTaskStartCallback"), task);
     }
 
+    private void jitterThreadStart(Task<?> task) {
```

Comment:
Great idea!

I wonder if we should make it non-static so it can be set by a test, and unset in its `tearDown` method. But let's go with static. For now, we can use it by setting the system property when doing `mvn clean install`. 

Once we've fixed (most of) the failures that it exposes, then we can think about enabling it for some tests in jenkins - for regression testing purposes. But we want jenkins to be deterministic!